crates/api_models/src/payments.rs

```diff
@@ -3230,3 +3229,57 @@ pub struct PaymentLinkDetails {
     pub max_items_visible_after_collapse: i8,
     pub sdk_theme: Option<String>,
 }
+
+#[derive(Clone, Debug, serde::Deserialize, ToSchema, serde::Serialize)]
+#[serde(deny_unknown_fields)]
+
+pub struct PaymentLinkListConstraints {
+    /// limit on the number of objects to return
+    pub limit: Option<i64>,
+
+    /// The time at which payment is created
+    #[schema(example = "2022-09-10T10:11:12Z")]
+    #[serde(default, with = "common_utils::custom_serde::iso8601::option")]
+    pub created: Option<PrimitiveDateTime>,
+
+    /// Time less than the payment created time
```

Comment:
```suggestion
    /// Time less than the payment link created time
```
I think everywhere it should be payment link